ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -575,7 +575,7 @@ export default function ViewQuote() {
           }
         }}
         submitText={t('swap')}
-        onCancel={async () => await dispatch(navigateBackToBuildQuote(history))}
```

Comment:
Is it necessary to remove the async and await? I'm not sure what the side effects might be, given that `navigateBackToBuildQuote` returns an async function

Reply:
The async/await is having no effect; if you hover over `await` in your text editor, the linter is saying that it in't necessary, thus the call isn't useful.